tests/e2e/greptimedbcluster/test_cluster_frontend_ingress.go

```diff
@@ -80,7 +80,7 @@ func TestClusterFrontendIngress(ctx context.Context, h *helper.Helper) {
 	err = h.RunHTTPTest("http://"+host+"/v1/sql", data, http.MethodPost)
 	Expect(err).NotTo(HaveOccurred(), "failed to run HTTP test")
 
-	data = "sql=SELECT * FROM numbers"
+	data = "sql=show databases"
```

Comment:
```suggestion
	data = "sql=SELECT 1"
```

show databases is disabled on cloud